static/tests/mailserver.js

```diff
@@ -13,7 +13,7 @@ describe('Whisper MailServer', () => {
 
     describe('Check prerequisites', () => {
         console.log('Expecting MailServer running.')
-        console.log('./build/bin/wnode-status -mailserver -passwordfile=./static/keys/wnodepassword -http -httpport 8540 -listenaddr=127.0.0.1:8549 -identity=./static/keys/wnodekey')
+        console.log('./build/bin/statusd -les=false -shh -shh.mailserver -passwordfile=./static/keys/wnodepassword -http -httpport 8540 -listenaddr=127.0.0.1:8549 -identity=./static/keys/wnodekey')
```

Comment:
So, `les` is `true` by default?

Reply:
Uff, I assumed so. But now I see that it's `false`. I think it should be enables as LES node is what most of `statusd` usages probably will be, what do you think?

Reply:
Yes, would vote for it as it hopefully will become a standard protocol for most clients anywhere.

Reply:
I would keep it as `false` by default actually. By default I expect to start as little protocols as possible and opt-in for the ones I need.

Reply:
@adambabik sorry, just saw your comment. Default behaviour for `statusd` is to run les. But we can change it, of course.